### PR TITLE
Add character radiation editing to debug menu [RDY]

### PR DESCRIPTION
**Pull request by ussdefiant60**
The debug menu can let you edit a lot of things about your character in the name of testing/cheating, but for whatever reason you can't get at the radiation level you're eating stat penalties/mutations from in there. This was annoying me, so i decided to figure out some C++ by adding it in.

As far as i can tell, its working, and i've shoved it into the Set Health section of Edit Player/NPC. I can only build on Windows non-tiles, though, so if anyone else wants to test this on Linux, feel free.

### CODE REVIEW

File: src/debug_menu.cpp [hunk 1]
```diff
@@ -344,6 +345,11 @@ void npc_edit_menu()
                     if( query_int( value, _( "Set the value to? Currently: %d" ), p.get_healthy_mod() ) ) {
                         p.set_healthy_mod( value );
                     }
+                case 2:
+                    if( query_int( value, _( "Set the value to? Currently: %d"), p.radiation ) ) {
```
Comment (BrettDong) [inline src/debug_menu.cpp]: astyle regression in this line

File: src/debug_menu.cpp [hunk 2]
```diff
@@ -344,6 +345,11 @@ void npc_edit_menu()
                     if( query_int( value, _( "Set the value to? Currently: %d" ), p.get_healthy_mod() ) ) {
                         p.set_healthy_mod( value );
                     }
```
Comment (BrettDong) [inline src/debug_menu.cpp]: Where is the `break;` statement of this block?

**Comment by ussdefiant60** [comment]
break added. I have no idea how to tell what astyle is complaining about, so i deleted a stray line break i saw in the area.

**Comment by ussdefiant60** [comment]
Kevin was kind enough to point out where our astyle settings are located so i could plug them in to code::blocks' formatter. Hopefully this will work

File: src/character.cpp [hunk 3]
```diff
@@ -1537,6 +1538,8 @@ void Character::set_healthy_mod(int nhealthy_mod)
 {
     healthy_mod = nhealthy_mod;
 }
+
+
```
Comment (codemime) [inline src/character.cpp]: Minor: these blank lines are irrelevant (here and below).

**Pull request merged**